examples/calling-apis/spa-with-backend-api/react-hono-langgraph/README.md

```diff
@@ -0,0 +1,168 @@
+# Chat SPA Application with Auth0 AI, LangGraph, and Federated Connection with Access Tokens
```

Comment:
LangGraph example added today. A few screenshots:

<img width="1069" height="956" alt="Screenshot 2025-08-11 at 7 35 35 PM" src="https://github.com/user-attachments/assets/2f90bf1d-3386-47a3-98b8-5d65d842798e" />

<img width="1047" height="1054" alt="Screenshot 2025-08-11 at 7 36 17 PM" src="https://github.com/user-attachments/assets/cfadfa97-6713-4274-9499-eabc415124a5" />


Key differences:
<img width="542" height="918" alt="Screenshot 2025-08-11 at 5 17 45 PM" src="https://github.com/user-attachments/assets/78751106-63b0-4d55-99bd-a35cb31fcb74" />

Interrupt flow and `GraphInterrupt`:
<img width="581" height="936" alt="Screenshot 2025-08-11 at 6 39 42 PM" src="https://github.com/user-attachments/assets/4db91ddf-e0d3-4e9c-a47d-b09e1e678f55" />